--- a/chrome/browser/ui/views/accelerator_table.cc
+++ b/chrome/browser/ui/views/accelerator_table.cc
@@ -164,6 +164,7 @@ const AcceleratorMapping kAcceleratorMap[] = {
     {ui::VKEY_N, ui::EF_PLATFORM_ACCELERATOR, IDC_NEW_WINDOW},
     {ui::VKEY_T, ui::EF_SHIFT_DOWN | ui::EF_PLATFORM_ACCELERATOR,
      IDC_RESTORE_TAB},
+    {ui::VKEY_K, ui::EF_PLATFORM_ACCELERATOR, IDC_DUPLICATE_TAB},
 
     // Alt by itself (or with just shift) is never used on Mac since it's used
     // to generate non-ASCII characters. Such commands are given Mac-specific
@@ -187,7 +188,7 @@ const AcceleratorMapping kAcceleratorMap[] = {
      IDC_FOCUS_INACTIVE_POPUP_FOR_ACCESSIBILITY},
     {ui::VKEY_D, ui::EF_ALT_DOWN, IDC_FOCUS_LOCATION},
     {ui::VKEY_E, ui::EF_CONTROL_DOWN, IDC_FOCUS_SEARCH},
-    {ui::VKEY_K, ui::EF_CONTROL_DOWN, IDC_FOCUS_SEARCH},
+    //{ui::VKEY_K, ui::EF_CONTROL_DOWN, IDC_FOCUS_SEARCH},
     {ui::VKEY_T, ui::EF_SHIFT_DOWN | ui::EF_ALT_DOWN, IDC_FOCUS_TOOLBAR},
     {ui::VKEY_RIGHT, ui::EF_ALT_DOWN, IDC_FORWARD},
     {ui::VKEY_HOME, ui::EF_ALT_DOWN, IDC_HOME},